.github/release-drafter.yml

```diff
@@ -0,0 +1,4 @@
+template: |
+  ## What’s Changed
+
+  $CHANGES
```

Comment:
We can probably use some of the cooler features of the latest version. Like using different PR labels to bump the major, minor versions.

heres one I made eariler https://github.com/Smirl/highheath/blob/master/.github/release-drafter.yml